## Tonser 2020.5.12

id: 0
init_pose_x: 0.0
init_pose_y: 0.0
takeoff_height: 1.5
circle_d: 2.0

#0: GPS,1: VICON,2: CAMERA
current_mode: 0

#0: SEND_NED_POSITION,1:SEND_NED_POSITION_RPY, 2:SEND_NED_RPY
current_state: 0

#0:LOCAL_POSITION_SP 1(不支持):GLOBAL_POSITION_SP 2:ATTITUDE_SP 3:VELOCITY_SP};
current_sp: 0

baudrate: 57600
serial_timeout: 10
rate: 30

serial_name: "/dev/ttyUSB0"

sub_pos_topic_name: "/vision_pose/pose" # 如果使用,请使用对init_pose处理之后的数据
use_out_sp: 0 #1 use
out_sp_name: ""
